packages/dataviews/package.json

```diff
@@ -20,6 +20,7 @@
 	},
 	"main": "build/index.js",
 	"module": "build-module/index.js",
+	"calypso:src": "src/index.ts",
```

Comment:
I don't know what this property is for, but `docs/monorepo` [mention it as required](https://github.com/Automattic/wp-calypso/blob/trunk/docs/monorepo.md#:~:text=Your%20package.json%20can%20have%20any%20of%20the%20normal%20properties%20but%20at%20a%20minimum%20should%20contain%20main%2C%20module%2C%20calypso%3Asrc%20and%20sideEffects.).

Reply:
This is for the Calypso build (the main app) to be able to consume source files in packages without first running the build step -- the one that creates the `build` or `dist` folders.

Calypso webpack build can work directly with the source (TS) files, it has all the TS/Babel transpilation set up. The `dist`/`build` directories are only for consumers that get the package from NPM.

Please add the same `calypso:src` field to the `exports`/`"."` definition a few lines below. We'll soon migrate the Calypso tooling setup to the more modern `exports` definition, in another PR.

Reply:
I pushed the suggested change.